07-routing-gossip.md

```diff
@@ -308,33 +308,31 @@ The receiving node:
     this `node_id`:
       - SHOULD queue the message for rebroadcasting.
       - MAY choose NOT to queue messages longer than the minimum expected length.
-  - SHOULD NOT connect to a node which has an unknown, _even_ `features` bit set
-  in the `node_announcement`.
   - MAY use `rgb_color` AND `alias` to reference nodes in interfaces.
-    - SHOULD insinuate their self-signed origin.
+    - SHOULD insinuate their self-signed origins.
 
 ### Rationale
 
-New node features are possible in the future; backwards compatible (or
-optional) ones will have odd feature bits, incompatible ones will have
-even feature bits. These may be propagated by nodes even if they
-can't use the announcements themselves.
+New node features are possible in the future: backwards compatible (or
+optional) ones will have _odd_ `feature` _bits_, incompatible ones will have
+_even_ `feature` _bits_. These may be propagated by nodes even if they
+cannot process the announcements themselves.
 
-New address types can be added in the future; as address descriptors have
+New address types may be added in the future; as address descriptors have
 to be ordered in ascending order, unknown ones can be safely ignored.
-Future fields beyond `addresses` can still be added, optionally with
-padding within `addresses` if they require certain alignment.
+Additional fields beyond `addresses` may also be added in the future—with
```

Comment:
Nit: "future – with" instead of "future–with"?

Reply:
I think this is a matter of preference, so far I've been using without spaces, can change it, but most important thing I think is to maintain consistency.

Reply:
Agreed. As long as it is consistent I'm fine :-)